### PR TITLE
QC-1153 Indicate that an object is a moving window in its title

### PR DESCRIPTION
**Pull request by knopers8**

### CODE REVIEW

**Comment by knopers8** [comment]
Just a comment for @Barthelemy and @justonedev1 . The reported window duration is the merged object validity. Usually it will not be equal to the cycle duration, because while objects from different EPNs will be subject to the same cycle duration, the cycles will be out of phase. This means, that the merged validity is going to be within `[cycleDuration, 2*cycleDuration]`.

In case that someone asks why, this is why.

File: Framework/src/MonitorObjectCollection.cxx [hunk 1]
```diff
@@ -120,6 +121,41 @@ const std::string& MonitorObjectCollection::getTaskName() const
...
+  if (hours > 0) {
+    result << hours << "h";
+  }
+  if (minutes > 0 || hours > 0) {
+    result << minutes << "m";
+  }
+  result << seconds << "s";
+  if (durationMs < 1000) {
+    result << durationMs << "ms";
+  }
```
Comment (Barthelemy) [inline Framework/src/MonitorObjectCollection.cxx:144]: the ms are ignored if the duration is more than 1 second, correct ?

**Review by Barthelemy — APPROVED** [review]

File: Framework/src/MonitorObjectCollection.cxx (hunk 1 above)
Comment (knopers8) [reply Framework/src/MonitorObjectCollection.cxx:144]: Correct. I don't think there is need for such verbosity if the validity is longer. It's just for shifters to be aware what they are looking at.

Comment (Barthelemy) [reply Framework/src/MonitorObjectCollection.cxx:144]: I agree

**Pull request merged**